### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by bhmdev**
Setup

### CODE REVIEW

File: src/main/frontend/src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react"
+const App = props => {
```
Comment (DanAlberts) [inline src/main/frontend/src/components/App.js:3]: You want a space between your imports and your code

File: src/main/frontend/src/main.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import App from "./components/App"
+ReactDOM.render(
```
Comment (DanAlberts) [inline src/main/frontend/src/main.js:5]: You want a space between your imports and your code here

File: src/main/resources/static/index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <title>Food Chain Reviews</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/foundation/6.2.4/foundation.min.css">
+    <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+<div id="app">
+</div>
+<script type="application/javascript" src="/bundle.js"></script>
+</body>
+</html>
```
Comment (DanAlberts) [inline src/main/resources/static/index.html]: This needs to be indented properly here.

**Review by DanAlberts — CHANGES_REQUESTED** [review]
Requested changes

**Review by bhmdev — COMMENTED** [review]
All changes made

**Review by DanAlberts — APPROVED** [review]

**Review by DanAlberts — APPROVED** [review]

**Pull request merged**